demos/switch.scss

```diff
@@ -27,10 +28,10 @@
 
 .demo-switch--custom {
   $color: $material-color-red-500;
-
-  @include mdc-switch-track-color($color);
-  @include mdc-switch-knob-color($color);
-  @include mdc-switch-focus-indicator-color($color);
+  
+  @include mdc-switch-toggled-on-track-color($color);
```

Comment:
Do you think it makes sense to create a new mixin which packages these three mixins?

`mdc-switch-toggle-color($color);`

Reply:
Yah, since the mixin applies color to only ON state it makes sense to name it `mdc-switch-toggled-on-color`.

Update the docs too.

Reply:
Done